src/chat/root/controller.js

```diff
@@ -1,9 +1,8 @@
 module.exports = class RootController {
-    constructor(LoginService) {
+    constructor(ChatService,LoginService) {
```

Comment:
Are you making use of ChatService here? If so, you should put a space after the comma.